### DISCUSSION TITLE
Error after updating plugins

### DISCUSSION THREAD

**Discussion by PH5HP** [rejetto/hfs#665] [General]
Hi,

I updated 2 plugins today. It seems one is generating an error. See screenshot.

Kind regards,
Henk - ph5hp.nl - fileserver.ph5hp.nl

![image](https://github.com/rejetto/hfs/assets/140949730/ed5be1af-0409-43d6-b71b-2e2dd0596d68)

**Comment by rejetto** [comment]
i see.
please try updating file-icons again

**Reply by PH5HP** [reply]
Hi,

TNX... I updated the file-icons plugin and it seems it has been solved. At the moment no errors anymore in the console.

Kind regards,
Henk - ph5hp.nl - fileserver.ph5hp.nl
